src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -134,6 +141,53 @@ class HaConfigDashboard extends LitElement {
                     .pages=${configSections.dashboard}
                   ></ha-config-navigation>
                 </ha-card>`}
+          <div class="tips">
+            <ha-svg-icon .path=${mdiLightbulbOutline}></ha-svg-icon>
+            <span class="tip-word">Tip!</span>
+            <span class="text"
+              >${this.hass.localize(
+                "ui.panel.config.tips.join",
+                "forums",
+                html`<a
+                  href="https://community.home-assistant.io/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Forums</a
+                >`,
+                "twitter",
+                html`<a
+                  href="https://www.home-assistant.io/twitter/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Twitter</a
+                >`,
+                "discord",
+                html`<a
+                  href="https://www.home-assistant.io/join-chat/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Chat</a
+                >`,
+                "blog",
+                html`<a
+                  href="https://www.home-assistant.io/blog/"
```

Comment:
Should we use the documentationUrl helper here, so you get to beta blog if you are on beta?

Reply:
There is a beta blog?

Reply:
Yeah, while beta is active, the release blog post is visible on rc.home-assistant.io. The documentationUrl helper handles this automatically.